.packit.yaml

```diff
@@ -33,7 +33,10 @@ jobs:
   - job: copr_build
     trigger: pull_request
     targets:
-      - epel-7-x86_64
+      epel-7-x86_64:
+        additional_repos:
+          - https://beaker-project.org/yum/server/RedHatEnterpriseLinux7/
+          - https://vault.centos.org/centos/7/extras/x86_64/
```

Comment:
Upstream mock configuration disables the extras repository, and there is no other way to include the CentOS repository for now. More reasons to migrate quickly.